devito/finite_differences/derivative.py

```diff
@@ -134,30 +135,32 @@ def __new__(cls, expr, *dims, **kwargs):
         obj._fd_order = fd_orders
         obj._deriv_order = orders
         obj._side = kwargs.get("side", centered)
-        obj._stagger = kwargs.get("stagger", tuple([left]*len(obj._dims)))
         obj._transpose = kwargs.get("transpose", direct)
         obj._eval_at = as_tuple(kwargs.get("eval_at"))
-
+        obj._x0 = kwargs.get('x0', {d: d for d in obj._dims})
```

Comment:
I think the attribute name "dims" should be turned into "dimensions" ASAP. For consistency.

Reply:
Not the point of this already big PR